docker-compose.yml

```diff
@@ -0,0 +1,22 @@
+version: '2'
+
+services:
+
+  traefik:
+    image: traefik:latest
+    command: --web --docker --docker.domain=docker.localhost --logLevel=DEBUG
+    ports:
+      - "80:80"
+      - "443:443"
+      - "8080:8080"
```

Comment:
bah oui, par contre il faudrait p-e restreindre depuis quelles ips


On Tue, Sep 6, 2016 at 11:42 PM, Guillaume Gerbaud <notifications@github.com
> wrote:

> In docker-compose.yml
> <https://github.com/gocaine/go-dart-deployment/pull/1#discussion_r77724109>
> :
>
> > @@ -0,0 +1,22 @@
> > +version: '2'
> > +
> > +services:
> > +
> > +  traefik:
> > +    image: traefik:latest
> > +    command: --web --docker --docker.domain=docker.localhost --logLevel=DEBUG
> > +    ports:
> > +      - "80:80"
> > +      - "443:443"
> > +      - "8080:8080"
>
> est ce vraiment utile ? (d'avoir accès au dashboard ? )
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/gocaine/go-dart-deployment/pull/1/files/3dfe954e2c870390c1e7e7ebbd27eac6258fd488#r77724109>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AChQZ3UWBOVdE76ZFnK9csbtwTcg90sVks5qnd44gaJpZM4J2UCW>
> .
>



-- 

*Mathieu POUSSE | **06.28.89.08.43** |* *@m_pousse*
*Zenika Rennes*
2 bis place Saint Melaine
35000 RENNES